packages/core/src/shared/extensionUtilities.ts

```diff
@@ -48,6 +48,13 @@ export function productName() {
     return isAmazonQ() ? 'Amazon Q' : `${getIdeProperties().company} Toolkit`
 }
 
+/** Gets the client name stored in oidc */
```

Comment:
potentially a dumb question, but what is "oidc"?

Reply:
I just stole this from one of the comments in the codebase 😄:

IAM Identity Center OpenID Connect (OIDC) is a web service that enables a client (such as CLI or a native application) to register with IAM Identity Center. The service also enables the client to fetch the user’s access token upon successful authentication and authorization with IAM Identity Center